src/fs.c

```diff
@@ -3,88 +3,88 @@
 #include <stdlib.h>
 #include <string.h>
 
+#include "cpu.h"
 #include "fs.h"
 #include "io.h"
-#include "cpu.h"
 #include "vc.h"
 
 typedef struct {
-    char    filepath[64];                   /* 0x0000 */
+    char filepath[64]; /* 0x0000 */
     union {
         struct {
-            char        filepath_old[64];   /* 0x0000 */
-            char        filepath_new[64];   /* 0x0040 */
-        } fsrename;                         /* 0x0080 */
+            char filepath_old[64]; /* 0x0000 */
+            char filepath_new[64]; /* 0x0040 */
+        } fsrename; /* 0x0080 */
```

Comment:
I noticed you've been moving the offset comments in front, should we do that here too?